nix/master.nix

```diff
@@ -680,6 +765,61 @@ in
         ++ lib.optional (cfg.outputsPath != null)
           # Allow buildbot-master to write to this directory
           "d ${cfg.outputsPath} 0755 buildbot buildbot - -";
+
+
+      services.oauth2-proxy = lib.mkIf (cfg.accessMode ? "fullyPrivate") {
```

Comment:
When this is fleshed out, it would be also nice to have something mentioned in the README or examples?

Reply:
Of course, I always leave that as last, since rewriting it 3 times is not fun.